### PR TITLE
adding missing band instance for std. map

### PR DESCRIPTION
**Pull request by vmarquez**
Thought this might be useful.

### CODE REVIEW

File: tests/src/test/scala/scalaz/MapTest.scala [hunk 1]
```diff
@@ -1010,6 +1010,7 @@ object MapTest extends SpecLite {
   checkAll(zip.laws[IntMap])
   checkAll(bifoldable.laws[==>>])
   checkAll(FoldableTests.anyAndAllLazy[IntMap])
+  checkAll(band.laws[Int ==>> ISet[Int]])
```
Comment (xuwei-k) [inline tests/src/test/scala/scalaz/MapTest.scala]: already exists here

https://github.com/scalaz/scalaz/blob/cd13e74e346b4da23d78bf109d5c3bdc77b0ac97/tests/src/test/scala/scalaz/MapTest.scala#L974

**Comment by vmarquez** [comment]
@xuwei-k oops I will remove that one.

**Comment by puffnfresh** [comment]
Isn't it only a Band when the value is one too? ISet has a Band, so passes test but looks like "String ==>> String" type checks but shouldn't have a Band instance right?

**Comment by vmarquez** [comment]
You are right @puffnfresh, I have fixed that.

**Comment by vmarquez** [comment]
@xuwei-k not sure why the build is failing, do I need to fix something else?

**Comment by xuwei-k** [comment]
fixed in https://github.com/scalaz/scalaz/commit/c3f2fd331ddc46eb700013f292d7e773efd93179
please rebase and squash commits

**Comment by vmarquez** [comment]
@xuwei-k thanks! I have merged and rebased again.

**Comment by xuwei-k** [comment]
merged https://github.com/scalaz/scalaz/commit/7a449affd22daeef953e9a2c11e3579fb19e13c4

**Pull request closed without merging**